app/slices/FlexibleContainer.scala

```diff
@@ -21,9 +21,13 @@ object FlexibleGeneral extends FlexibleContainer {
     val numOfStandard = stories.size - numOfSplash
     val defaultStandardStoryLimit = 8
 
-    val standardStoryLimit = collectionConfigJson.displayHints
-      .flatMap(_.maxItemsToDisplay)
-      .getOrElse(defaultStandardStoryLimit)
+    val standardStoryLimit = collectionConfigJson match {
+      case Some(config) =>
+        config.displayHints
+          .flatMap(_.maxItemsToDisplay)
+          .getOrElse(defaultStandardStoryLimit)
+      case None => defaultStandardStoryLimit
+    }
```

Comment:
You've used two different approaches to handling `Option`s here, pattern matching and `flatMap`. It is typically easy to read code that sticks to one approach.

```suggestion
    val standardStoryLimit = collectionConfigJson
      .flatMap(_.displayHints)
      .flatMap(_.maxItemsToDisplay)
      .getOrElse(defaultStandardStoryLimit)
```